webpack.config.js

```diff
@@ -1,52 +1,52 @@
 const path = require('path');
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const HtmlWebpackInjectPreload = require('@principalstudio/html-webpack-inject-preload');
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlBundlerPlugin = require('html-bundler-webpack-plugin');
 const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
 const TerserPlugin = require("terser-webpack-plugin");
 const Dotenv = require('dotenv-webpack');
 const svgToMiniDataURI = require("mini-svg-data-uri");
 
 module.exports = (env) => {
   return {
-    entry: './src/index.js',
```

Comment:
Using the bundler plugin the entry point is a HTML template, see the `entry` option in the plugin